imagetest/test_suites/README.md

```diff
@@ -11,64 +11,52 @@ new GCE image.
 
 Tests are broken down by suite below:
 
-## Test suites ##
+## Test Suites
 
-### Test suite: cvm ###
+### Test suite: cvm
 
-#### TestCVMEnabled ####
+#### TestSEVEnabled
```

Comment:
Can you change this to TestSEVEnabled/TestSEVSNPEnabled/TestTDXEnabled? I also think we should be more explicit about what the test is actually doing, which is making sure the instance can boot with the respective confidential instance type set in the config and that it can load it's guest kernel module.

Reply:
Done.